windshaft/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM ubuntu:vivid
+
+MAINTAINER Azavea
+
+RUN apt-get update
+
+# Windshaft requires node < 11; vivid has 10.25 in nodejs package
+RUN apt-get -q install -y nodejs npm git-core \
```

Comment:
One of the Docker best practices is to combine similar steps into a single `RUN` instruction. If you switch to the `node` base image, these can probably be combined as:

```dockerfile
RUN apt-get update && apt-get install -y --no-install-recommends \
    libcairo2-dev \
    libpango1.0-dev \
    libjpeg8-dev \
    libgif-dev \
    libpq-dev \
    rm -rf /var/lib/apt/lists/*
```